### PR TITLE
Bug fix prop name

### PR DESCRIPTION
**Pull request by dariajung**
Issue #21

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -148,6 +153,10 @@ export class Minifier {
     var newName = chars.join('');
     if (this.checkReserved(newName)) {
       return this.generateNextPropertyName(newName);
+      // Property names cannot start with a number. Generate next possible property name that starts
+      // with the first alpha character.
+    } else if (this.isNumeric(chars[0])) {
```
Comment (mprobst) [inline src/main.ts]: I think it's cleaner here to test for [0-9], you don't really want to parse that as a number.

**Comment by mprobst** [comment]
I would try and not get too hung up on the whole `generateNextPropertyName` thing; if it's easier just don't use `0-9` in your alphabet - it's not really that useful to fiddle with this before we can measure end-to-end minification performance (gzipped result from minifying e.g. Angular with this). IMHO it's more interesting to spend some time on the externs/internal symbols/etc topic (and don't forget your talk!).

File: src/main.ts (hunk 1 above)
Comment (rkirov) [inline src/main.ts]: +1 JS supports inequality tests on strings using lexicographic ordering

**Comment by rkirov** [comment]
LGTM, @mprobst I asked Daria to simply add numerics and '$', '_', assuming it shouldn't be too complicated. Lets merge this and move on.

Daria, if more bugs appear in the generation, feel free to reduce the alphabet if easier that fixing the bug.

File: src/main.ts (hunk 1 above)
Comment (dariajung) [inline src/main.ts]: Switched to regex

**Pull request closed without merging**